spring-integration-azure/src/main/java/com/microsoft/azure/spring/integration/storage/queue/StorageQueueTemplate.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+
+package com.microsoft.azure.spring.integration.storage.queue;
+
+import com.microsoft.azure.spring.integration.storage.queue.factory.StorageQueueFactory;
+import com.microsoft.azure.storage.StorageException;
+import com.microsoft.azure.storage.queue.CloudQueue;
+import com.microsoft.azure.storage.queue.CloudQueueMessage;
+import org.springframework.util.Assert;
+
+
+public class StorageQueueTemplate implements StorageQueueOperation {
+    private final StorageQueueFactory storageQueueFactory;
+
+    StorageQueueTemplate(StorageQueueFactory storageQueueFactory) {
+        this.storageQueueFactory = storageQueueFactory;
```

Comment:
Why does this constructor has only default access modifiers?

Reply:
I should add public modifier.